README.md

```diff
@@ -7,7 +7,7 @@ Cronitor-client is a simple Java library designed to help monitor your app. With
 Simply add this line in your pom.xml file :
 ```
 <dependency>
-    <groupId>ca.sm360.cronitor</groupId>
+    <groupId>io.cronitor</groupId>
     <artifactId>cronitor-client</artifactId>
     <version>1.0.1</version>
```

Comment:
If I understood well, it will be the 1.1.0 now after the move on your Maven repository